### ISSUE TITLE
candidate pages with a photo throw NotImplementedError: This backend doesn't support absolute paths.

### ISSUE THREAD

**Issue by symroe** [DemocracyClub/yournextrepresentative#654]
https://sentry.io/democracy-club-gp/ynr-v9/issues/704105907/

```
NotImplementedError: This backend doesn't support absolute paths.
(30 additional frame(s) were not displayed)
...
  File "debug_toolbar/panels/templates/panel.py", line 56, in _request_context_bind_template
    yield
  File "popolo/models.py", line 613, in get_display_image_url
    return get_thumbnail(self.primary_image.path, "x64").url

NotImplementedError: This backend doesn't support absolute paths.
```

**Comment by chris48s** [comment]
e.g:

* https://candidates.democracyclub.org.uk/person/49111
* https://candidates.democracyclub.org.uk/person/33882
* https://candidates.democracyclub.org.uk/person/25189

This won't have come up in manual or unit testing for Image/ImageExtra merge because it will have been fine with the local filesystem storage engine

**Comment by symroe** [comment]
Closed in #655
